README.md

```diff
@@ -30,11 +30,7 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
 
 You can run the tests locally with: `bundle exec rake`.
 
-The publishing API includes [Pact contract tests](docs/pact_testing.md), which verify that the service behaves in the way expected by its clients. The tests the *consumer driven contract testing* pattern. You can run these on their own using:
-
-```sh
-$ bundle exec rake pact:verify
-```
+The Publishing API also has [contract tests with GDS API Adapters](https://docs.publishing.service.gov.uk/manual/pact-testing.html) and [with Content Store](https://docs.publishing.service.gov.uk/manual/pact-testing.html#special-cases-and-tech-debt), where it acts as the contract "consumer". Read the guidance for how to run the tests locally.
```

Comment:
Should "the guidance" be a link? I'm somewhat guessing it's meant to be one of the earlier links in this paragraph but that seems a leap? I guess it may be respective?